exposed-core/build.gradle.kts

```diff
@@ -12,5 +13,7 @@ dependencies {
     api(kotlin("stdlib"))
     api(kotlin("reflect"))
     api("org.jetbrains.kotlinx", "kotlinx-coroutines-core", Versions.kotlinCoroutines)
+    api("org.jetbrains.kotlinx", "kotlinx-serialization-json", Versions.kotlinxSerialization)
+    api("org.postgresql", "postgresql", Versions.postgre)
```

Comment:
PostgreSQL DB returns [a special type](https://jdbc.postgresql.org/documentation/publicapi/org/postgresql/util/PGobject.html) `PGobject` for deserialization. I didn't figure out a way to catch the type that doesn't involve including this specific dependency.